### ISSUE TITLE
Docs: Change Cheyenne references to Derecho

### ISSUE THREAD

**Issue by samsrabin** [ESCOMP/CTSM#2223]

**Comment by slevis-lmwg** [comment]
@linniahawkins I'm starting to work on this, so I'm adding myself to the assignees.

**Comment by slevis-lmwg** [comment]
Changes here affect python files, so I ran these tests before opening the PR and will likely rerun before merging to b4b-dev:
- [x] make black and lint
- [x] ./run_ctsm_py_tests -u and -s
- [x] `./run_sys_tests -s clm_pymods -c ctsm5.4.025 --skip-generate`

**Comment by linniahawkins** [comment]
Sorry for dropping the ball on this - let me know if there is anything I can follow up on.

**Comment by slevis-lmwg** [comment]
No worries @linniahawkins :-)  Would you be willing to review my work in the PR? I will request a review, and you can decide. 

If you do review the PR, you will see specific questions that I asked Erik. Feel free to weigh in if you can answer any of them.

**Comment by linniahawkins** [comment]
Certainly, I haven't done a PR review before is there anything I should know?

**Comment by slevis-lmwg** [comment]
> Certainly, I haven't done a PR review before is there anything I should know?

If you click the [Files Changed button in the PR](https://github.com/ESCOMP/CTSM/pull/3838/changes) it will show you what I changed in all my commits. If you have comments, you hover over the line of interest, click on the blue plus sign, and type. You can do independent comments or "start a review" and in the latter case you will need to click on the button that completes the review when you're ready.
